assets/xml/objects/object_bubble.xml

```diff
@@ -0,0 +1,6 @@
+<Root>
+    <File Name="object_bubble" Segment="6">
+        <Texture Name="gBubbleTex" Format="ia8" Width="64" Height="64"/>
```

Comment:
Looks good but try to be careful here since this might conflict with gameplay_keep for example

Reply:
As well as possibly conflicting with EnBb. The enemy's in-game name is Shabom, so we may want to use that here.